deploy/chart/templates/0000_50_olm_09-operatorgroup.crd.yaml

```diff
@@ -70,6 +76,7 @@ spec:
             staticProvidedAPIs:
               type: boolean
               description: If true, OLM will not modify the OperatorGroup's providedAPIs annotation.
+          additionalProperties: false
```

Comment:
`additionalProperties: false` isn't allowed in crd validation: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation